nautobot/circuits/templates/circuits/circuittype_retrieve.html

```diff
@@ -26,7 +26,7 @@
             <div class="panel-heading">
                 <strong>Circuits</strong>
             </div>
-            {% include 'inc/table.html' with table=circuits_table %}
+            {% include 'inc/table.html' with detail_view=True table=circuits_table %}
```

Comment:
If this is something app detail views will need to adopt in order to have correct rendering, let's make sure we document it thoroughly.

What are the cases where we are using `inc/table.html` *without* `detail_view=True`? If those are less common and/or less-commonly-used-by-apps, maybe we could invert the default here?

Reply:
invert the setting in the latest commit

Reply:
The app detail view wouldn't need to do anything to adopt this change I believe